docs/user-guide/logging/stackdriver.md

```diff
@@ -20,149 +24,125 @@ fluentd-cloud-logging-kubernetes-node-0f64     1/1       Running   0          32
+Stackdriver logging agents attaches metadata to each log entry, for you later
+to use in queries to select only messages you're interested in, e.g. from
+a particular pod.
 
-![Cloud Logging Console](/images/docs/cloud-logging-console.png)
+The most important pieces of metadata are the resource type and log name.
+Resource type of container logs is `container` and it's also named
+`GKE Containers` in the UI (even if the Kubernetes cluster is not on GKE).
+Log name is the name of the container, so that if you have a pod with
+two containers, named in specs `container_1` and `container_2`, their logs
+will have log names `container_1` and `container_2` respectively.
 
-When we view the logs in the Developer Console we observe the logs for both invocations of the container.
+System components have different resource type, `compute`, named
+`GCE VM Instance` in the interface. Log names for system components are fixed.
+For a GKE node, every log entry from a system component has one the following
+log names:
 
-![Both Logs](/images/docs/all-lines.png)
+* docker
+* kubelet
+* kube-proxy
 
-Note the first container counted to 108 and then it was terminated. When the next container image restarted the counting process resumed from 0. Similarly if we deleted the pod and restarted it we would capture the logs for all instances of the containers in the pod whenever the pod was running.
-
- Logs ingested into Stackdriver Logging may be exported to various other destinations including [Google Cloud Storage](https://cloud.google.com/storage/) buckets and [BigQuery](https://cloud.google.com/bigquery/). Use the Exports tab in the Cloud Logging console to specify where logs should be streamed to. You can also follow this link to the
- [settings tab](https://pantheon.corp.google.com/project/_/logs/settings).
-
- We could query the ingested logs from BigQuery using the SQL query which reports the counter log lines showing the newest lines first:
-
-```shell
-SELECT metadata.timestamp, structPayload.log
-FROM [mylogs.kubernetes_counter_default_count_20150611]
-ORDER BY metadata.timestamp DESC
-```
-
-Here is some sample output:
-
-![BigQuery](/images/docs/bigquery-logging.png)
-
-We could also fetch the logs from Google Cloud Storage buckets to our desktop or laptop and then search them locally. The following command fetches logs for the counter pod running in a cluster which is itself in a Compute Engine project called `myproject`. Only logs for the date 2015-06-11 are fetched.
+You can learn more about viewing logs on [the dedicated Stackdriver page](https://cloud.google.com/logging/docs/view/logs_viewer).
 
+One of the possible ways to view logs in using [gcloud](https://cloud.google.com/sdk/gcloud/)
```

Comment:
a) Change "in using" to "is using the"
b) Change "[gcloud](https://cloud.google.com/sdk/gcloud/)" to "[gcloud logging](https://cloud.google.com/logging/docs/api/gcloud-logging)"
c) Change "from a Google Cloud SDK" to "from the Google Cloud SDK).